### ISSUE TITLE
Please don't squat names on crates.io

### ISSUE THREAD

**Issue by kornelski** [yuulive/asdfdf#2]
I've noticed you're forking projects and grabbing 2-letter names on crates.io. Please don't do that.

**Comment by faresbakhit** [comment]
Yes very strange behavior and unacceptable. Forks with no addition but a name change is misleading to the original maintained crate.
